crates/flightrepl/src/lib.rs

```diff
@@ -109,7 +109,7 @@ pub async fn run(repl_config: ReplConfig) -> Result<(), Box<dyn std::error::Erro
                 println!("\nAny other line will be interpreted as a SQL query");
                 continue;
             }
-            "show tables" => {
+            "show tables" | "show tables;" => {
```

Comment:
Why not just trim space and ;

Reply:
to escape from rust ownership check, you would need to do multiple lines to get rid of multiple suffix, etc. 

Which is not as nice as this one.